app/assets/stylesheets/navigation.scss

```diff
@@ -0,0 +1,41 @@
+/************** NAVIGATION *******************************/
+
+#nav-container {
+  border-bottom: 1px solid #d9d9d9;
+  box-shadow: 2px 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.dropdown {
+  border: 1px solid #d9d9d9;
+  border-top: none;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.dropdown li > a {
```

Comment:
Merge rule `.dropdown li > a` with rule on line 8<br>Selector should have depth of applicability no greater than 2, but was 3